muxassign/main.c

```diff
@@ -104,7 +193,7 @@ void create_core_assignment(const char *core, char *sys, char *rom, int cache, e
             break;
         }
         case PARENT: {
-            delete_files_of_type(core_dir, "cfg", NULL, 1);
+            delete_files_of_type(core_dir, "/core.cfg", NULL, 1);
```

Comment:
previously this was deleting all cfg files but this would break favourites from working until you ran the game at least once.  now it only deletes the core.cfg files and uses the methods above to update existing game cfg files with the new core assignment so favourites will continue to work.